2019-12-07 19:27:00,144 manager INFO 1.1.0
2019-12-07 19:27:00,144 manager INFO Copyright (C) 2003-2010
2019-12-07 19:27:00,144 manager INFO   Noriaki Ando
2019-12-07 19:27:00,144 manager INFO   Intelligent Systems Research Institute, AIST
2019-12-07 19:27:00,144 manager INFO Manager starting.
2019-12-07 19:27:00,144 manager INFO Starting local logging.
2019-12-07 19:27:00,151 manager.namingmanager INFO NameServer connection succeeded: corba/localhost
2019-12-07 19:27:02,161 ManagerServant INFO Master manager not found
2019-12-07 19:27:02,164 manager WARNING Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\OpenRTM_aist\Manager.py", line 1900, in initManagerServant
    reffile = file(self._config.getProperty("manager.refstring_path"),'w')
IOError: [Errno 2] No such file or directory: '/var/log/rtcmanager.ref'

2019-12-07 19:27:02,165 LocalServiceAdmin INFO All the local services are enabled.
2019-12-07 19:27:02,174 manager INFO Component type conf file: Change.conf loaded.
2019-12-07 19:27:02,180 ec_base INFO setRate(1000000.000000) done
2019-12-07 19:27:02,180 ec_base INFO setRate(1000.000000) done
2019-12-07 19:27:02,181 rtobject INFO 1 execution context was created.
2019-12-07 19:27:02,187 rtobject INFO Initial active configuration set is default-set.
2019-12-07 19:27:06,874 SdoServiceAdmin WARNING No available SDO service in the factory: 
2019-12-07 19:27:06,874 SdoServiceAdmin ERROR type  already exists.
2019-12-07 19:27:06,875 SdoServiceAdmin WARNING No available SDO service in the factory: IDL:OpenRTM/ComponentObserver:1.0
2019-12-07 19:27:06,875 SdoServiceAdmin ERROR type 3935e35e-08cd-4298-a122-73f8c47bf460 already exists.
2019-12-07 19:27:18,404 rtobject ERROR Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\OpenRTM_aist\RTObject.py", line 1560, in on_execute
    ret = self.onExecute(ec_id)
  File "C:\workspacess\Change\Change.py", line 128, in onExecute
    rawdata = rawdata.replace('\n','')
AttributeError: 'list' object has no attribute 'replace'

2019-12-07 19:27:23,121 ec_worker ERROR State of the RTC is not ACTIVE_STATE.
